dotCMS/src/main/java/com/dotcms/contenttype/model/field/Field.java

```diff
@@ -244,4 +241,33 @@ public FieldValueRenderer valueRenderer() {
   public FieldValueRenderer listRenderer() {
     return new FieldValueRenderer() {};
   }
+
+  /**
+   * Field' properties to be set by the UI
+   * @return
+   */
+  public Collection<ContentTypeFieldProperties> getFieldContentTypeProperties(){
+    return null;
+  }
+
+  /**
+   * Key for the Field' help text
+   * @return
+   */
+  @JsonIgnore
+  public String getContentTypeFieldHelpTextKey(){
+    String legacyName = LegacyFieldTypes.getLegacyName(this.getClass());
+    return "field.type.help." + legacyName;
+  }
+
+  /**
+   * Key for the Field'label
+   * @return
+   */
+  @JsonIgnore
+  public String getContentTypeFieldLabelKey(){
+    String legacyName = LegacyFieldTypes.getLegacyName(this.getClass());
+    String key = legacyName.substring(0, 1).toUpperCase() + legacyName.substring(1);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "key". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)